### PR TITLE
feat: bio use v-html

### PR DESCRIPTION
**Pull request by ches4117**

### CODE REVIEW

File: packages/karbon/src/module.ts [hunk 1]
```diff
@@ -444,6 +444,10 @@ const karbon = defineNuxtModule<ModuleOptions>({
       './runtime/plugins/storipress',
       './runtime/plugins/storipress-payload',
       './runtime/plugins/custom-field',
+      {
+        src: './runtime/plugins/entities.client',
+        mode: 'client',
+      },
```
Comment (DanSnow) [inline packages/karbon/src/module.ts:450]: missing server plugin

**Review by DanSnow — CHANGES_REQUESTED** [review]

File: packages/karbon/src/module.ts (hunk 1 above)
Comment (ches4117) [reply packages/karbon/src/module.ts:450]: 好

Comment (ches4117) [reply packages/karbon/src/module.ts:450]: fixed

File: packages/karbon/src/module.ts [hunk 2]
```diff
@@ -278,25 +278,25 @@ const karbon = defineNuxtModule<ModuleOptions>({
...
-            })
-          } else {
-            return genObjectFromRaw({
-              presetFactory: config.importName,
-              options: JSON.stringify(config.options),
-            })
-          }
-        }),
-      )}
+          resolvedSEO.map((config) => {
```
Comment (ches4117) [inline packages/karbon/src/module.ts]: 咦, 又重新排版了

Comment (ches4117) [reply packages/karbon/src/module.ts]: fixed

**Review by DanSnow — CHANGES_REQUESTED** [review]
Author page doesn't have `bioHTML` 
image

**Review by DanSnow — APPROVED** [review]

**Pull request merged**